plugins/rest-api-endpoints/routes.json

```diff
@@ -7695,6 +7695,7 @@
       "url": "/repos/:owner/:repo/comments/:comment_id"
     },
     "getCommitRefSha": {
+      "deprecated": "\"Get the SHA-1 of a commit reference\" will be removed. Use \"Get a single commit\" instead with medaType format \"sha\" instead.",
```

Comment:
@gr2m I think it should read `mediaType` instead of `medaType`.

Any hint how to get the SHA-1 of a commit reference using `repos.getCommit`? According to the documentation for [`repos.getCommit`](http://octokit.github.io/rest.js/#octokit-routes-repos-get-commit), `sha` is required, which doesn't make it possible to use it to get the `sha` when you don't already have it.

Reply:
Could you make an issue or send a PR to fix the typo?

This should work

```js
octokit.repos.getCommit({
  owner,
  repo,
  commit_sha,
  mediaType: {
    format: 'sha'
  }
})
```

`commit_sha` can be any kind of git reference, such as a branch name, it’s misleading right now. I’ve raised the issue with the docs team.

Reply:
Thanks for your help!

Done: https://github.com/octokit/rest.js/pull/1381